integration-tests/tests/circuits.rs

```diff
@@ -5,7 +5,14 @@ macro_rules! declare_tests {
             async fn [<serial_test_evm_ $name>]() {
                 log_init();
                 let block_num = GEN_DATA.blocks.get($block_tag).unwrap();
-                let pk = None;
+                let pk = if $real_prover { Some((*EVM_CIRCUIT_KEY).clone()) } else { None };
+                let (builder, _) = gen_inputs(*block_num).await;
+                let mut block = block_convert::<Fr>(&builder.block, &builder.code_db).unwrap();
+                block.circuits_params.max_evm_rows = 0;
```

Comment:
It's `0` by default. And here we have a default `circuits_params` right? So this statement is unneded.

Reply:
Also ,if we want to reuse the PK/VK, we should set a fixed value instead of compute at runtime.

Reply:
Yes, this statement is a left over, and perhaps it is causing the errorsm thanks

Reply:
Except it is not 0 by default in the integration tests. I set it to zero to trigger the calculation of the minimum rows, but then I created the method that does that without setting any parameters.

Anyway, it is an error, and might cause an error

Reply:
Ohh by default i meant the `Default` trait impl for `CircuitsParams`.

Reply:
I think in integration tests is where we need to set `MAX_EVM_ROWS` instead of `0`.

Reply:
Yes, that is done already in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1148/files#diff-a5f41858194d9e4113ec4124717d29358cca4df88def6d33815a7ab20205e3d1R57

Before I set it to zero there to obtain the calcualted number of rows, but with the new method https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1148/files#diff-b4e5f73a1df679165a0103a62054eeff42c1bcbba32eff0ba8287c52740f2738R214-R223 that was not necessary and could actually caused problems